backend/src/server/context.ts

```diff
@@ -38,4 +38,14 @@ export const getUser = (context: Context): dbUser => {
   throw new Error('No user given in context!')
 }
 
+export const getClientTimezoneOffset = (context: Context): number => {
+  if (
+    (context.clientTimezoneOffset || context.clientTimezoneOffset === 0) &&
+    Math.abs(context.clientTimezoneOffset) <= 27 * 60
```

Comment:
Do we have timezone offsests greater than 24h or is there a buffer of three hours included?